lib/aliquot/payment.rb

```diff
@@ -66,6 +66,8 @@ def process
 
       begin
         @message = JSON.parse(decrypt(aes_key, @signed_message[:encryptedMessage]))
+        @message.merge!('threedsCryptogram' => @message.delete('3dsCryptogram')) if @message['3dsCryptogram']
```

Comment:
You could do `@message.transform_keys!('3dsCryptogram' => 'threedsCryptogram')` to rename the key and avoid the conditional

Reply:
I have to reverse this as it appears that it is not supported in ruby 2.7

Reply:
My bad, it was introduced in 2.5, but looks like it only worked with a block then.